packages/react-components/src/components/chart/trendCursor/series/useHandleSeries.ts

```diff
@@ -44,5 +44,5 @@ export const useHandleSeries = ({
     };
 
     delayedRender({ updateFunction: update });
-  }, [chartRef, series.length, setGraphic]);
+  }, [chartRef, series, setGraphic]);
```

Comment:
why the change from series.length?

Reply:
We change the actual series when we add the hidden property. To trigger the re-render of all the graphics we need to check that the series changed.

Reply:
im worried that this will cause rerenders on the trendcursors every time anything in the series changes including styles (color, line wdith, etc.) which shouldnt be triggering a rerender